Add upcoming event + reformat all events

dominiquesydow opened the pull request:
## Description

Add new event and reformat all events.

## TODOs

- [x] Add Nov 19, 2020 event 
- [x] Reformat all events so that focus is on title + speakers
- [x] Add Discord mention
- [x] Check rendering with `hugo serve`

## Questions

None

## Status

- [x] Ready to go

jaimergp commented:
LGTM!

Do you want to add a mention for the Discord server? I wouldn't post a public invite link because that can get too spammy, but maybe mentioning that it exists and that you can request an invite over Twitter or email?

dominiquesydow commented:
I had thought of it as well but dropped it because I did not want to post the URL - but I like your suggestion (email/Twitter) and added a section about this.

I also changed the section headings to
- Stay up-to-date via our mailing list
- Socialize on our Discord server
- Join us for our upcoming seminars
- Check out our previous seminars

Do you like this or is it too verbose?

jaimergp commented:
Go ahead!
